zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/event/ClusterEvent.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.cluster.event;
+
+/**
+ * Cluster Event
+ */
+public enum ClusterEvent {
+  CREATE_INTP_PROCESS
```

Comment:
Why other events not in this enum ? Like NEW_NOTE、DEL_NOTE、REMOVE_NOTE_TO_TRASH and etc.

Reply:
Other types, Will be added in the next PR.
https://issues.apache.org/jira/browse/ZEPPELIN-3778